cclient/serialization/json/get_inclusion_states.c

```diff
@@ -13,6 +13,12 @@ retcode_t json_get_inclusion_states_serialize_request(get_inclusion_states_req_t
   retcode_t ret = RC_ERROR;
   char const *json_text = NULL;
   log_debug(json_logger_id, "[%s:%d]\n", __func__, __LINE__);
+
+  if (!req->tips && !req->transactions) {
```

Comment:
Aren't `tips` optional ? 
https://docs.iota.org/docs/iri/0.1/references/api-reference#getinclusionstates

Reply:
optional, but need the json array.